src/main/java/uk/gov/hmcts/reform/ucmc/model/ClaimValue.java

```diff
@@ -2,11 +2,13 @@
 
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import lombok.RequiredArgsConstructor;
 
 @Data
 @Builder
 @RequiredArgsConstructor
+@NoArgsConstructor(force = true)
```

Comment:
Have you tried constructor with @JasonCreator annotation on top of that? 
```suggestion
@JsonCreator
    public ClaimValue(Integer lowerValue, Integer higherValue) {
        this.lowerValue = lowerValue;
        this.higherValue = higherValue;
    }
```

Reply:
With lombok it will be: `@RequiredArgsConstructor(onConstructor = @__(@JsonCreator))`

Reply:
Easy either way - just thought the whole point was to avoid having to explicitly declare the constructor which can get quite verbose the more fields you have in your builder...?

Reply:
Just tried suggestion above but unfortunately: 

`java.lang.IllegalArgumentException: Invalid type definition for type `uk.gov.hmcts.reform.ucmc.model.ClaimValue`: Argument #0 has no property name, is not Injectable: can not use as Creator [constructor for uk.gov.hmcts.reform.ucmc.model.ClaimValue, annotations: {interface com.fasterxml.jackson.annotation.JsonCreator=@com.fasterxml.jackson.annotation.JsonCreator(mode=DEFAULT)}]
 at [Source: UNKNOWN; line: -1, column: -1]`